dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -105,6 +105,7 @@ public int createPullReq(GHRepository origRepo, String branch,
         try {
             GHPullRequest pullRequest = origRepo.createPullRequest(title, forkRepo.getOwnerName() + ":" + branch,
                     origRepo.getDefaultBranch(), body);
+            pullRequest.setLabels("dependencies");
```

Comment:
can we run it against an org/repo that doesn't have **dependencies** label already created? Not sure If the will auto-create a new one  If it doesn't exist. In the earlier version of GHE, I believe we had to create a label explicitly first and then apply it. 
LGTM otherwise

Reply:
As I have seen from other tools, the label would be auto-created.
I read https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#about-labels and didn't see anything about the label needing to exist first.

Reply:
wanted to be sure about it. vaguely remember this was needed a couple of years back. 
Tested it on test-temp/test/pull/1 on soma. works fine. we should be good here